azure-pipelines.yml

```diff
@@ -0,0 +1,15 @@
+jobs:
+  - template: common-pipelines.yml
+    parameters:
+      name: Linux
+      vmImage: ubuntu-1604
+
+  - template: common-pipelines.yml
+    parameters:
+      name: Windows
+      vmImage: vs2017-win2016
```

Comment:
how we pick the vm image? is there any other choice that may solve the known issue?

Reply:
Ehhhhhh.... Only there are three vm images for windows (vs2017-win2016,win1803,vs2015-win2012r2). I've tried them all before but the issue still exists.

Reply:
See the link for env. https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=vsts&tabs=yaml